package-lock.json

```diff
@@ -4090,6 +4089,9 @@
       },
       "engines": {
         "node": ">= 10"
+      },
+      "optionalDependencies": {
+        "@commitlint/load": ">6.1.1"
```

Comment:
Please remove the changes from package-lock.

Reply:
Okay, done.